app/models/organization.rb

```diff
@@ -39,4 +40,32 @@ def next_reg_number_seq
     reg_number_seq
   end
 
+  def is_web_url
+     begin  
+      w = URI.parse(web)
+      rescue  
+      return false
+    end 
+    return true
+  end
+
+  def ensure_url
+    if !is_web_url
+      return web
+    end
+    if (URI.parse(web).class == URI::HTTP || URI.parse(web).class == URI::HTTPS) 
```

Comment:
Line is too long. [81/80]<br>Don't use parentheses around the condition of an `if`.<br>Trailing whitespace detected.